.github/dependabot.yml

```
@@ -14,13 +14,19 @@ updates:
     - "justeattakeaway/justsaying-maintainers"
   open-pull-requests-limit: 10
   ignore:
+  - dependency-name: AWSSDK.Extensions.NETCore.Setup
+    versions:
+    - "> 3.7.300"
```

We can just simplify this to "always ignore" by removing the versions.

```suggestion
  - dependency-name: AWSSDK.Extensions.NETCore.Setup
```